challenge-1.js

```diff
@@ -2,6 +2,21 @@
 
 function fizzBuzz(number) {
   // Write your code here
+  let resolution = [];
+  
+  for (let i = 1; i < number+1; i++) {
+      if(i % 3 === 0 && i % 5 === 0) {
+          triage = 'FizzBuzz'
+        } else if(i % 3 === 0){
+          triage = 'Fizz'
+        } else if(i % 5 === 0) {
+          triage = 'Buzz'
+        } else {
+          triage = i
+        }
```

Comment:
🚀  We could use a external function to return the string direct in the triage variable, to avoid the use of multiples else if ❤️